private/scripts/helpers.sh

```diff
@@ -1,5 +1,46 @@
 #!/usr/bin/env bash
 
+# Defines some global variables for colors.
+normal=$(tput sgr0)
+bold=$(tput bold)
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+yellow=$(tput setaf 3)
+# Additional styles we aren't currently using.
+# italic=$(tput sitm)
+#blue=$(tput setaf 4)
+#magenta=$(tput setaf 5)
+#cyan=$(tput setaf 6)
+
+# Main function that runs the script.
+function main() {
+  help_msg="Usage: bash ./private/scripts/helpers.sh <command>
+  Available commands:
+    install_sage: Install Sage.
+    maybe_create_symlinks: Create a symlinks to WordPress files, if they don't already exist."
+
+  if [ -z "$1" ]; then
+    echo "${red}No command specified.${normal}"
+    echo "${help_msg}"
+    exit 1;
+  fi
+
+  if [ "$1" == "help" ]; then
+    echo "${help_msg}"
+    exit 0;
+  fi
+
+  # Check for a valid command.
+  if [ "$1" != "install_sage" ] && [ "$1" != "maybe_create_symlinks" ]; then
+    echo "${red}Invalid command specified.${normal}"
+    echo "${help_msg}"
+    exit 1;
+  fi
+
+  # Execute the command passed.
+  "$@"
```

Comment:
would prefer to check that `$1` is a valid function name before executing. optional. Also, if none of the functions have arguments of their own, could use `$1` instead of `$@`

Reply:
>would prefer to check that $1 is a valid function name before executing

That's handled inside `main`: https://github.com/pantheon-systems/wordpress-composer-managed/blob/5bb5db58b4da9b2f137f5fea6bc95fa7a5822a69/private/scripts/helpers.sh#L22-L38

Reply:
ah overlooked this